.changeset/empty-swans-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+"thirdweb": patch
+---
+
+Expsose GaslessConfigs type
```

Comment:
Could you include a code snippet or type examples here? When we expose previously internal functions of types we have to treat it like its a net new feature (because it is from the public API POV)